backend/README.md

```diff
@@ -0,0 +1,192 @@
+}
+```
+
+## /user/login
+
+```
+curl --location --request POST 'http://localhost:4080/user/login' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+	"email": "name@domain.com",
+	"password": "yourpass"
+}'
+```
+**Returns**
+```
+{
+    "token": "eyJhbGciOiJIUzI1NiIs....R-sJ9iHNUde0"
+}
+```
+
+## /user/me
+
+```
+curl --location --request GET 'http://localhost:4080/user/me' \
+--header 'x-access-token: eyJhbGci....blSXAPm0'
+
+```
+**Returns**
+```
+{
+    "agreeToBeNotified": false,
+    "userType": 1,
+    "createdAt": "2020-04-02T10:16:12.090Z",
+    "_id": "5e85bf05420bac7bcfece08f",
+    "username": "username",
+    "email": "name@domain.com",
+    "password": "$2a$10$21w/RANQMJoc2Ge6FpcSSOCpY1S2ae6li5dv5xNeQEzewphkneGcS",
+    "idUser": 1,
+    "encryptedUserId": "5f6451160f76aac8a02493787dc940a0057746c6401cc49757664ce1032b8450",
+    "__v": 0
+}
+```
+
+`encryptedUserId` returns the encrypted email as user id.
+
+# Report
+
+## POST: /report 
+
+This let to add a result test to the report table.
+
+```
+curl --location --request POST 'http://localhost:4080/report' \
+--header 'x-access-token: eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyIjp7ImlkIjoiNWU4ODhhZDBlZGEwN2U2ZGFlYjY0ODBiIn0sImlhdCI6MTU4NjAxNjA2MiwiZXhwIjoxNTg2MDE5NjYyfQ.hmxFFAz1P80Yq4Q2iA6D8IpCOhI5_7xkfZYWDkQOHK4' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+	"idUser": "'9365df4e9acef8b63b45dc3534491225ac32630abe6991f6bf5a74c9803412fc'",
+	"testDate": "03/02/2020",
+	"testResult": 0
+}'
+```
+**Returns**
+
+```
+{
+    "report": {
+        "createdAt": "2020-04-04T16:04:00.725Z",
+        "_id": "5e88b01316714c664f5ef11e",
+        "idUser": 1,
+        "testDate": "2020-03-02T03:00:00.000Z",
+        "testResult": 0,
+        "idReport": 13,
+        "__v": 0
+    }
+}
+```
+
+## GET: /report/{idUser} 
+
+Get the list of reported tests
+
+
+```
+curl --location --request GET 'http://localhost:4080/report/9365df4e9acef8b63b45dc3534491225ac32630abe6991f6bf5a74c9803412fc' \
+--header 'x-access-token: eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyIjp7ImlkIjoiNWU4ODhhZDBlZGEwN2U2ZGFlYjY0ODBiIn0sImlhdCI6MTU4NjI4Njg4MiwiZXhwIjoxNTg2MjkwNDgyfQ.zVNwkDZCXtlxRJnDhonptMxbgpngrB3T7cNIy8vde_I'
+```
+**Returns**
+
+```
+{
+    "reports": [
+        {
+            "createdAt": "2020-04-07T19:24:31.388Z",
+            "_id": "5e8cd38dc43d8007408e600d",
+            "idUser": "9365df4e9acef8b63b45dc3534491225ac32630abe6991f6bf5a74c9803412fc",
+            "testDate": "2020-03-04T03:00:00.000Z",
+            "testResult": 1,
+            "idReport": 17,
+            "__v": 0
+        },
+        {
+            ...
+        }
+    ]
+}
+```
+
+# Migrations
+source: https://www.npmjs.com/package/migrate-mongo
+
+# Mongo Commands
+## Mongo CSV import example
+docker-compose exec mongo mongoimport --host mongo --db myappdb --collection ciudads --type csv --fields idProvincia,idCiudad,ciudad,ciudadCorta --file /data/db/CodCiudad.csv
```

Comment:
It would be better to include here a relevant example here.

Reply:
I will remove it. Int this version it is not necessary because we haven't data to import.